### ISSUE TITLE
Spring break development plan, please reply with your commitment availability 

### ISSUE THREAD

**Issue by baiyz** [billlipeng/ECEN689-602-Spring2015#7]
STMF429 Hardware Platform: York
emWin GUI: York
ST IMU Sensor: Aaron + Zoy
Stepper Driver: Billy + Zoy
Simulink Model: Yiren
Assembly Design: Zoy
Matlab Embedded Coder: Yiren + Zoy
GitHub Coordinator: Zoy

Development retreat from March 14 - 20, place: WERC 102

**Comment by yorkbeary** [comment]
Agreed. maybe I could touch a little on the stepper and IMU, on the lower level. It's called hardware abstraction.
